website/views.py

```diff
@@ -4515,3 +4519,59 @@ def submit_pr(request):
         return render(request, "submit_pr.html")
 
     return render(request, "submit_pr.html")
+
+
+# Global variable to store the vector store
+vector_store = None
+
+
+@api_view(["POST"])
+def chatbot_conversation(request):
+    # Rate Limit Check
+    utc_now = datetime.now(timezone.utc)
+    today = utc_now.date()
+    rate_limit_key = f"global_daily_requests_{today}"
+    request_count = cache.get(rate_limit_key, 0)
+
+    if request_count >= 10:
```

Comment:
Could you extract it to a separate variable to avoid adding another magic number? BTW, is it supposed to be a global or per user daily limit?

Reply:
it's global as donnie want